cvat/apps/dataset_manager/bindings.py

```diff
@@ -1307,8 +1308,9 @@ def get_frame(task_id: int, idx: int) -> ProjectData.Frame:
                 anno_manager.to_shapes(
                     task.data.size,
                     included_frames=task_included_frames,
+                    deleted_frames=set(f for t_id, f in self.deleted_frames if t_id ==  task.id),
```

Comment:
why not `task_data.deleted_frames.keys()`?

Reply:
This is the same